index.bs

```diff
@@ -559,31 +583,56 @@ Width of the play-area bounds in meters. The bounds are defined as an axis-align
 <dfn attribute for="VRStageParameters">sizeZ</dfn>
 Depth of the play-area bounds in meters. The bounds are defined as an axis-aligned rectangle on the floor. The center of the rectangle is at (0,0,0) in standing-space coordinates. These bounds are defined for safety purposes. Content should not require the user to move beyond these bounds; however, it is possible for the user to ignore the bounds resulting in position values outside of this rectangle.
 
-
-## Navigator Interface extension ## {#interface-navigator}
+## VR Interface ## {#interface-vr}
 
 <pre class="idl">
-partial interface Navigator {
-  Promise&lt;sequence&lt;VRDisplay&gt;&gt; getVRDisplays();
-  readonly attribute FrozenArray&lt;VRDisplay&gt; activeVRDisplays;
+interface VR : EventTarget {
+  Promise&lt;sequence&lt;VRDisplay&gt;&gt; getDisplays();
+  readonly attribute FrozenArray&lt;VRDisplay&gt; activeDisplays;
```

Comment:
This doesn't make sense to me. I'm unclear of the use case, and it's ambiguous.

See my proposed `navigator.vr.referringDisplay(s)` idea: https://gist.github.com/cvan/9c0a2e6e1a6839816128aeb461eb3184#file-sample-usage-js-L21-L32

<hr>

À la [`navigator.languages`](https://developer.mozilla.org/en-US/docs/Web/API/NavigatorLanguage/languages)), I suggest having introducing a **`navigator.vr.preferredDisplays`, an array of display name strings for user-preferred VR headsets**.

If we think of VR devices/displays à la languages/locales, you can imagine a user flow something like this:

1. When a user first plugs in a headset, the UA remembers that headset name in some internal persisted list of "known VR displays" in that user's UA profile.
2. A user opens the UA's "Preferences" to add/remove/rearrange the preferred headsets to use when display WebVR content.
3. When a requests _any_ page, the browser now starts sending a `Accept-VR-Display` **request header** (e.g., `Accept-VR-Display: HTC Vive, Oculus Rift`)  – à la the [`Accept-Language`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Content_negotiation#The_Accept-Language_header) request header [e.g., `Accept-Language: en-US, fr; q=0.7`]).
4. If a page's **response header** is `Content-VR-Display: Gear VR, HTC Vive` (or a `<meta>` tag: `<meta http-equiv="content-vr-display" content="available HTC Vive, Oculus Rift">` – [or [CSP-Style](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP): `<meta http-equiv="Content-VR-Policy" content="default-display 'HTC Vive'; supported-displays 'HTC Vive', 'Oculus Rift'; prefetch-navigation-to 'https://*.neocities.org'">`]), then the UA knows which displays are supported by this WebVR page (or scenes within the page, if we're going to get complicated).
5. The UA now knows that when the page loads, the user may want to *enter VR* by clicking on some in-content button that calls `<VRDisplay>.requestPresent(…)` (where `<VRDisplay>` would be a VR Display that exists in `navigator.vr.preferredDisplays` and is also supported by the content creator of the WebVR scene).

Here's what I mocked up in Firefox's Preferences, for example:

<img width="1552" alt="screen shot 2017-01-26 at 5 50 52 pm" src="https://cloud.githubusercontent.com/assets/203725/22359416/e1ce046a-e3fb-11e6-8516-89ed5ee3e7a3.png">
<img width="1552" alt="screen shot 2017-01-26 at 6 32 47 pm" src="https://cloud.githubusercontent.com/assets/203725/22359421/e5635e54-e3fb-11e6-8c7f-69636ba81d01.png">


<hr>

If while a WebVR source is presenting to a headset _and_ a window navigation occurs, the UA can ought to *fire the `navigate` event on `navigator.vr` _in addition to_ pushing the display to `navigator.vr.referringDisplays` (i.e., `attribute FrozenArray<VRDisplay> referringDisplays`).

A developer could then check that and immediately request presentation back to the display the user was using before/during the page navigation.